content/src/logic/deployments.ts

```diff
@@ -52,10 +53,14 @@ export async function retryFailedDeploymentExecution(
         )
       } catch (error) {
         // it failed again, override failed deployment error description
-        const errorDescription = error.message
-        components.failedDeploymentsCache.reportFailure({ ...failedDeployment, errorDescription })
+        const errorDescription = error.message + ''
 
-        logger.info(`Failed to fix deployment of entity`, { entityId, entityType })
+        // TODO [mendez] this condition has no test coverage
+        if (!errorDescription.startsWith(IGNORING_FIX_ERROR)) {
+          components.failedDeploymentsCache.reportFailure({ ...failedDeployment, errorDescription })
```

Comment:
So, if the deployment is ignored it shouldn't be reported as a failure, right? It should be removed from the failed deployments instead.

Reply:
Yes, that happens inside the deployEntity function